CONTRIBUTING.md

```diff
@@ -0,0 +1,104 @@
+# Contributing to PatternFly-ng
+
+We would love for you to contribute to PatternFly-ng and help make it even better than it is
+today!  Below is information on how to get started as a contributor to PatternFly-ng.
+
+ - [Code of Conduct](#coc)
+ - [Question or Problem?](#question)
+ - [Issues and Bugs](#issue)
+ - [Submission Guidelines](#submit)
+ - [Coding Rules](#rules)
+ - [Building the Code](#coding)
+ - [Releasing](#releasing)
+
+## <a name="coc"></a> Code of Conduct
+An open community is important to us, please help us maintain that by following our [Code of Conduct][coc].
+
+## <a name="question"></a> Do you have a question?
+There are a few things that can be done to resolve any questions or problems you might have.
+ - Search our [GitHub issues][github-issues]
+ - Join our patternfly-ng channel on [Slack](http://slack.patternfly.org)
+ - Join our mailing-list following the instructions on [patternfly.org](http://www.patternfly.org/community/)
+
+
+## <a name="issue"></a> Found a bug or issue?
+Please help us out if you find a bug by submitting an issue to our [GitHub Repository][github-issues].
+We also welcome [PR's][github-pr] if you'd like to submit a fix.
+
+## <a name="submit"></a> Submission guidelines
+The following should be included in all PR's
+ - A clear commit message describing what has been done
+ - A reference to the [issue][github-issues] being addressed (whether it's a bug being fixed or a new component being added, a GitHub issue should be added to track it).
+ - A unit test covering the included behavior
+ - Documentation:
+  - Add a new examples folder within the component folder if it doesn't already exist
+  - Add an example component following the style of one of the existing components.  
+  - Examples should include a working sample including the ability to toggle configuration values to demonstrate the full functionality of a component
+  - Links to the example code and generated typedoc documenation should be included following the example below
+ ``` angular2html
```

Comment:
this doesn't appear to be recognized by the prettifyer - can it be changed to just ```html ?